test.js

```diff
@@ -1,5 +1,12 @@
 var t = "hello";
 
-if (t === "hello') {
+var selec = '[data-dikkemik]';
+
+
+if (t === 'hello') {
     console.log('ok');
 }
+
+if (true) {
+  var sjaakDeman = 'false';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["sjaakDeman" is defined but never used](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2482812805/issues/source?bid=3260976&fileBranchId=3261113#l11)